src/ErrorPage.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/macro'
+import Button from './components/Button'
+
+function ErrorPage({ setErrorMessage, showMainPage }) {
+    function handleClick() {
+        setErrorMessage('')
+        showMainPage()
+    }
+
+    return (
+        <ErrorPageStyled>
+            <section className="county-class-red result-wrapper">
+                <h3>Sorry, Daten konnten nicht geladen werden.</h3>
+                <span>Probiere eine neue Suche!</span>
```

Comment:
You may want to set the styling in "result-wrapper" class to get rid of the `span`?

Reply:
Good point